content/en/docs/reference/glossary/host-aliases.md

```diff
@@ -14,4 +14,4 @@ tags:
 
 <!--more-->
 
-[HostAliases](/docs/reference/generated/kubernetes-api/v1.13/#hostalias-v1-corev) is an optional list of hostnames and IP addresses that will be injected into the Pod's hosts file if specified. This is only valid for non-hostNetwork Pods.
+[HostAliases](https://kubernetes.io/docs/reference/generated/kubernetes-api/{{< latest-version >}}/#hostalias-v1-core) is an optional list of hostnames and IP addresses that will be injected into the Pod's hosts file if specified. This is only valid for non-hostNetwork Pods.
```

Comment:
I'd use:
```suggestion
[HostAliases](https://kubernetes.io/docs/reference/generated/kubernetes-api/{{< param "version" >}}/#hostalias-v1-core) is an optional list of hostnames and IP addresses that will be injected into the Pod's hosts file if specified. This is only valid for non-hostNetwork Pods.
```

so that if `{{< latest-version >}}` refers at some point to v1.19, the v1.18 docs still signpost readers to the v1.18 API docs.

Reply:
Got it! I'm gonna change that.